tests/integration/annotation_import/conftest.py

```diff
@@ -173,6 +178,34 @@ def annotations_by_data_type(polygon_inference, rectangle_inference,
     }
 
 
+@pytest.fixture
+def annotations_by_data_type_v2(
```

Comment:
Separated from export v1 annotation mappings, cleaner also it was breaking exports v1